packages/protobuf/scripts/protobuf-patches/TxOutputType.js

```diff
@@ -25,10 +25,20 @@ export type TxOutputType =
           orig_index?: number,
           payment_req_index?: number,
       }
+      | {
+        address: string,
+        address_n?: typeof undefined,
+        script_type?: ChangeOutputScriptType,
+        amount: UintType,
+        multisig?: MultisigRedeemScriptType,
+        orig_hash?: string,
+        orig_index?: number,
+        payment_req_index?: number,
+    }
     | {
           address?: typeof undefined,
           address_n?: typeof undefined,
-          amount: '0',
+          amount: '0' | 0,
```

Comment:
@martykan  could you please just link where this change was needed? I presume it was again some test..

Reply:
Here https://github.com/trezor/trezor-suite/blob/76edaf1729905e275e4aeaf26164042f25890356/packages/connect/e2e/__fixtures__/signTransactionTaproot.ts#L139-L143

Reply:
sounds reasonable